lib/guardian/user_guardian.rb

```diff
@@ -30,6 +30,10 @@ def can_see_notifications?(user)
     is_me?(user) || is_admin?
   end
 
+  def can_see_drafts?(user)
+    is_me?(user) || is_admin?
```

Comment:
I think we need a spec for this as well :) Never know when we may regress.

Reply:
@tgxworld do you have a example of what this spec should look like?

Reply:
Something like https://github.com/discourse/discourse/blob/master/spec/components/guardian_spec.rb#L38-L42